app.rb

```diff
@@ -8,7 +16,9 @@ def list_all_music_albums
   end
 
   def list_all_movies
-    raise StandardError, 'not implemented'
+    @movies.each do |movie|
+      puts "date: #{movie.can_be_archived?}, silent: #{movie.silent}"
```

Comment:
This function is perfectly fine but it would be good if you display the date also.